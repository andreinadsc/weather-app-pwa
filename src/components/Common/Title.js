import classes from './Title.module.css';

const Title = ({ title, isSubtitle = false }) =>
    <h4 className={isSubtitle ? `${classes.header} ${classes.subtitle}` : classes.header}>{title}</h4>;

export default Title;