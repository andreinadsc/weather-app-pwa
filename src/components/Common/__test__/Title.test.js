import { render } from '@testing-library/react'
import Title from '../Title';

describe('Title Common Component', () => {
    test('should render header title correctly', () => {
        const { baseElement } = render(<Title title='Header' />)

        expect(baseElement).toMatchSnapshot();
    });

    test('should render subtitle correctly', () => {
        const { baseElement } = render(<Title title='Subtitle' isSubtitle={true} />)

        expect(baseElement).toMatchSnapshot();
    });
});