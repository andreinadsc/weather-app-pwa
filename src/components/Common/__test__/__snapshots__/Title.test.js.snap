// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Common Component should render header title correctly 1`] = `
<body>
  <div>
    <h4
      class="header"
    >
      Header
    </h4>
  </div>
</body>
`;

exports[`Title Common Component should render subtitle correctly 1`] = `
<body>
  <div>
    <h4
      class="header subtitle"
    >
      Subtitle
    </h4>
  </div>
</body>
`;
